public/app/features/connections/tabs/ConnectData/ConnectData.test.tsx

```diff
@@ -49,8 +49,14 @@ describe('Badges', () => {
     expect(screen.queryByText('Angular')).not.toBeInTheDocument();
   });
 
-  test('shows angular badge for angular plugins', async () => {
+  test('does not show angular badge for angular plugins because we do not load them', async () => {
```

Comment:
Should we remove this test-case? (As far as I understand the logic it was testing got removed?)

Reply:
I've removed it plus the test above that was testing more angular badge logic that's been removed.